hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/handlers/CryptoHandlerTestBase.java

```diff
@@ -103,32 +106,45 @@ public class CryptoHandlerTestBase {
             .build();
     protected static final long defaultAutoRenewPeriod = 720000L;
     protected static final long payerBalance = 10_000L;
+    protected AutoRenewConfig config = new AutoRenewConfig(Set.of("CONTRACT"));
```

Comment:
We should use config to get and create it

Reply:
Changed. Thanks